### PR TITLE
chore(bazel): add MODULE.bazel files for bzlmod

### PR DESCRIPTION
**Pull request by mmorel-35**
This provide a MODULE.bazel to be compatible with bzlmod
See [Bzlmod Migration Guide](https://bazel.build/external/migration) for more information

### CODE REVIEW

File: BUILD.bazel [hunk 1]
```diff
@@ -64,21 +65,23 @@ cc_library(
         # "src/simplex",
         # "src/test",
         # "src/util",
-        "bazel-bin"],
+        "bazel-bin",
+    ],
+    linkopts = ["-lpthread"],
```
Comment (mmorel-35) [inline BUILD.bazel:70]: I added this for https://github.com/bazelbuild/bazel-central-registry/pull/1940 to pass the presubmit validations

**Comment by galabovaa** [comment]
Looks good to me! Thank you @mmorel-35

**Comment by galabovaa** [comment]
Do you have any comments or suggestions, @Mizux

File: MODULE.bazel [hunk 2]
```diff
@@ -0,0 +1,17 @@
+module(
+    name = "highs",
+    version = "0.0.0",
```
Comment (Mizux) [inline MODULE.bazel]: should version be 1.7.0 (or updated later once everything seems to pass in the bazel central repository side ?)

File: MODULE.bazel [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    name = "bazel_skylib",
+    version = "1.3.0",
+)
+bazel_dep(
+    name = "rules_cc",
+    version = "0.0.9",
+)
+bazel_dep(
+    name = "zlib",
+    version = "1.2.11",
```
Comment (Mizux) [inline MODULE.bazel]: should try to update to 1.3.1 there is some CVE out there..
see: https://github.com/google/or-tools/issues/4211

File: MODULE.bazel [hunk 4]
```diff
@@ -0,0 +1,17 @@
+module(
+    name = "highs",
+    version = "0.0.0",
+)
+
+bazel_dep(
+    name = "bazel_skylib",
+    version = "1.3.0",
```
Comment (Mizux) [inline MODULE.bazel]: `1.6.1` is out
ref: https://github.com/bazelbuild/bazel-skylib/releases

File: MODULE.bazel (hunk 2 above)
Comment (mmorel-35) [reply MODULE.bazel]: This is usually updated at release time. The bcr bot can do that , take a look at it’s page if you are interested : https://github.com/bazel-contrib/publish-to-bcr

Comment (Mizux) [reply MODULE.bazel]: I'm more concerned with a user fetching the repo and having 0.0.0 as version while it is (at least) 1.7.0

**Pull request closed without merging**